src/write.c

```diff
@@ -152,7 +152,7 @@ void avifCodecEncodeOutputDestroy(avifCodecEncodeOutput * encodeOutput)
 typedef struct avifEncoderItem
 {
     uint16_t id;
-    uint8_t type[4];                      // 4-character 'item_type' field in the 'infe' (item info) box
+    uint8_t type[4];                      // 4-character 'item_type' field in the 'infe' (item info entry) box
```

Comment:
I noticed this while comparing the SlimHEIF proposal with libavif (src/write.c).